requirements.txt

```diff
@@ -1,9 +1,5 @@
-# Developers: Please do not modify this auto-generated file. If
```

Comment:
Why is this message removed? It is fine to change 3.7 to 3.8 but the message at the top should still be here.

Reply:
The requirements.txt file is auto generated by pip compile based on requirements.in file. So eventually it will get overwritten when you first launch oppia. I actually generated this file seperately to update the previous one which was generated for python 3.7 . I think the upper text is to be added manually as a warning for developers to not modify this file directly and let it be autogenerated throught pip compile and requirements.in .

Reply:
Ohh !! Let me just look into it . Maybe it should be autogenerated with it .

Reply:
ok yes it should write the above message before baking the requirements into the file. I will modify it asap. Thanks for the review @seanlip